pkg/source/chirpstack/source.go

```diff
@@ -148,12 +147,13 @@ func (p *Source) ExportDevice(devEui string) (*ttnpb.EndDevice, error) {
 	}
 
 	// Identifiers
-	if err := dev.EndDeviceIdentifiers.DevEui.UnmarshalText([]byte(devEui)); err != nil {
+	dev.Ids.DevEui, err = unmarshalWithTool(&types.EUI64{}, devEui)
```

Comment:
Shouldn't `dev.Ids` be initialized somewhere?

Reply:
Yes they should, fixed [here](https://github.com/TheThingsNetwork/lorawan-stack-migrate/compare/e868846a713c59862fb05e4ec97d09b598c76bc9..208da6ed7e5c02b9de41f52c293123830bd2f0cf) and sorted the values.